cachify.php

```diff
@@ -72,13 +72,21 @@
 	)
 );
 
+/* WP-CLI */
+add_action(
+    'init',
```

Comment:
Please, change the hook to `cli_init` (introduced in [WP-CLI 2.0](https://make.wordpress.org/cli/2018/08/08/wp-cli-v2-0-0-release-notes/)). The `cli_init` action fires only in WP-CLI context, so:

1. The `if ( defined( 'WP_CLI' ) && WP_CLI )` check from `Cachify_CLI::add_commands()` method can be dropped.
1. The new `inc/cachify_cli.class.php` file will not be autoloaded on every request in regular (non-cli) context.

Reply:
@chesio I updated the pull request

- hook changed to cli_init
- `if ( defined( 'WP_CLI' ) && WP_CLI )` check removed